examples/binaries/calc-hash/over-blocks/src/code.rs

```diff
@@ -0,0 +1,120 @@
+use crate::Method;
+use gstd::{exec, msg, ActorId, BTreeMap, Decode, Encode, MessageId, Vec};
+use shared::PackageId;
+use types::Package;
+
+#[no_mangle]
+unsafe extern "C" fn init() {
+    state::THRESHOLD = Some(msg::load().expect("Invalid threshold."));
+}
+
+#[no_mangle]
+unsafe extern "C" fn handle() {
+    unsafe {
```

Comment:
Have I missed it, why `unsafe` block inside an unsafe function?

Reply:
oh sry, converted from async fn